src/components/cart-card/CartCard.jsx

```diff
@@ -0,0 +1,87 @@
+import "./cart-card.css";
+import { useUser } from "../../context/user-context";
+import { useCart } from "../../context/cart-context";
+import axios from "axios";
+
+const CartCard = ({ image, title, price, includeStock, _id, qty }) => {
+  console.log(qty);
+  const { encodedToken } = useUser();
+  const { dispatch } = useCart();
+
+  const incrementItem = async () => {
+    // dispatch({
+    //   type: "INCREMENT_ITEM",
+    //   payload: { dispatch, _id, encodedToken },
+    // });
```

Comment:
Remove unnecessary commented code

Reply:
- I am still figuring out about this problem of navbar not sticking to bottom, will do in future. 
- The commented actually needs to be debugged in future that's why I have not removed it. Will remove in upcoming release.